src/components/NoticiaForm.jsx

```diff
@@ -24,12 +25,18 @@ class NoticiaForm extends Component {
     title: '',
     editorState: '',
     rawContent: '',
+    images: [],
     error: {
       message: '',
     },
     isLoading: false,
   }
 
+  // Uploads
+  storagePath = 'uploads'; // path in the storage
```

Comment:
should we be sub dividing different upload types. aka `uploads/noticias`?

Reply:
I don't think so. At least for now I can't see a reason to do it.